experimental/dds/tree2/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -268,7 +268,12 @@ const valueRebaser: FieldChangeRebaser<ValueChangeset> = {
 		}
 		if (change.value !== undefined) {
 			assert(revision !== undefined, 0x591 /* Unable to revert to undefined revision */);
-			inverse.value = { revert: reviver(revision, 0, 1)[0], revision };
+			inverse.value = {
+				revert: reviver(revision, 0, 1)[0],
+				// KLUDGE: Value field changes don't have local IDs so we use an arbitrary value.
+				// TODO: Either delete the value field implementation or give its changes some local IDs.
+				changeId: { revision, localId: brand(42) },
```

Comment:
Did you mean to fix this?

Reply:
No, I'm just banking on Justus deleting the Value FieldKind in the next few days